everestrs/everestrs-build/jinja/types.jinja2

```diff
@@ -10,7 +10,7 @@ pub struct {{ object.name }} {
 {% for p in object.properties %}
 /// {{ p.description | replace("\n", " ") }}
 #[serde(rename="{{ p.name }}"{% if p.data_type.extra_serde_annotations %},{{ p.data_type.extra_serde_annotations | join(",") }}{% endif %})]
-pub {{ p.name | snake }}: {{ p.data_type.name }},
+pub {{ p.name | identifier }}: {{ p.data_type.name }},
```

Comment:
Alternatively we can prepend/append to the variable names "something" like
```
pub var_{{ p.name | snake }}: {{ p.data_type.name }}
```

This would have the advantage that we can keep the naming "consistent" for the user and simplify the logic (we don't need then to check if its a reserved key word)

Reply:
As discussed on slack, both solutions are warty, let's go with the raw keywords and see if it sucks, we can always change.